documentation/modules/upgrading/proc-upgrade-brokers-newer-kafka.adoc

```diff
@@ -31,9 +31,7 @@ For the `Kafka` resource to be upgraded, check that:
 kubectl edit kafka _my-cluster_
 ----
 
-. If the `log.message.format.version` and `inter.broker.protocol.version` of the _current_ Kafka version are _the same_ as the new Kafka version, go to step 3.
```

Comment:
@scholzj - I thought we could remove this as the version does tend to change and we say keep it at the previous version anyway

Reply:
I think the new version you have there makes sense to me. So 👍 .